index.js

```diff
@@ -12,45 +12,79 @@ server.get('/', (req, res) => {
 
 //MVP Required endpoints
 
+// //POST - add new users to db
+server.post('/api/users', (req, res) => {
+      
+    db.insert(req.body)
+        .then(user => {
+            if ((req.body.name < 1) || (req.body.bio < 1)) {
+                res.status(400).json({ errorMessage: 'Please provide name and bio for the user.' })
+            } else {
+            res.status(201).json(user);
+            }
+        })
+        .catch(err => {
+            console.log('Error adding user', err);
+        })
+})
+
 //GET - see whole user list
 server.get('/api/users', (req, res) => {
     db.find()
         .then(users => {
             res.status(200).json(users)
         })
-    .catch(err => {
-        console.log('GET /user error')
-        res.status(500).json({ errorMessage: 'User information could not be retrieved.'})
-    })
+        .catch(err => {
+            console.log('GET /user error', err)
+            res.status(500).json({ errorMessage: 'The users information could not be retrieved.' })
+        })
 });
 
 //GET- find individual user
 server.get('/api/users/:id', (req, res) => {
     db.findById(req.params.id)
```

Comment:
Code is looking clean and readable. Good work. You can destructure your req.params.id to something like this 

const id = { req.params.id}

and then pass it into your findById function.

other than that good job!